lib/protoboeuf/decorated_field.rb

```diff
@@ -8,7 +8,16 @@ class DecoratedField
 
   extend Forwardable
 
-  def_delegators :@original_field, :name, :label, :type_name, :type, :number, :options, :oneof_index, :has_oneof_index?
+  def_delegators :@original_field,
+    :name,
+    :label,
+    :type_name,
+    :type,
+    :number,
+    :options,
+    :oneof_index,
+    :has_oneof_index?,
+    :json_name
```

Comment:
I know these are basically the same order as they were before the change, but now that I see them all on one line it irks me a bit that there's sort ordering to them. I'd make them alphabetic or lexicographic if the ordering is otherwise insignificant.